src/core/compact_object.h

```diff
@@ -445,13 +446,22 @@ class CompactObj {
     };
   } __attribute__((packed));
 
+  struct JsonConsT {
+    JsonType* json_ptr;
+    // Owning and must be explicitly deallocated
+    MiMemoryResource* mr;
```

Comment:
We don't really need this but I want to keep the DCHECKS. If in the future we decide we need an extra `8` bytes for `JsonConsT`, we can remove this :man_dancing: